modules/installation-and-upgrade/pages/connect-payg.adoc

```diff
@@ -7,13 +7,13 @@ In the three major public cloud providers (AWS, GCP and Azure), SUSE:
 * operates per-region RMT Servers mirroring repositories for products available as {payg}
 
 This document describes how to connect existing {payg} instance to {productname} server, and gives basic information about credentials collection from the instance.
-The goal of the connection is to extract authentication data so {productname} server is able to connect to cloud RMT host.
+The goal of the connection is to extract authentication data so {productname} server is able to connect to cloud RMT host. That will give the {productname} server access to products that are on the RMT host and that are not already available via the {scc} organization credentials.
 
 Before using {payg} feature make sure that: 
 
 * the {payg} instance is launched from the correct SUSE product image (e.g. SLES, SLES for SAP, SLE HPC) to allow access to the desired repositories
 * {productname} server has connectivity to the {payg} instance (ideally in the same region) either directly or via a bastion
-* valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products
+* valid {scc} credentials are entered in menu:Admin[Setup Wizard > Organization Credentials], even they provide only limited access to products - your subscription should at least offer access to the {productname} client tools.
```

Comment:
* Enter valid {scc} credentials under menu:Admin[Setup Wizard > Organization Credentials], your subscription offers access to the {productname} client tools.

Reply:
No. We are speaking of a minimum content of the credentials. The client tools are a requisite so it works.